nucliadb_utils/src/nucliadb_utils/authentication.py

```diff
@@ -66,7 +69,17 @@ async def authenticate(self, request: HTTPConnection) -> Optional[Tuple[AuthCred
 
         if self.user_header in request.headers:
             user = request.headers[self.user_header]
-            nuclia_user: BaseUser = NucliaUser(username=user)
+            nuclia_user: NucliaUser = NucliaUser(username=user)
+
+            raw_security_groups: Optional[str] = request.headers.get(self.security_groups_header)
+
+            security_groups: Optional[list[str]] = None
+            if raw_security_groups is not None:
+                security_groups = raw_security_groups.split(";")
```

Comment:
This parsing implies `;` is not a valid character for security groups, but we're not enforcing it in the API models

Reply:
Security models are defined here: https://github.com/nuclia/nucliadb/blob/main/nucliadb_models/src/nucliadb_models/security.py

Reply:
OR i can send the value in JSON with a proper list, what do you think?